lib/src/sv_common.c

```diff
@@ -158,6 +158,20 @@ sign_or_verify_data_free(sign_or_verify_data_t *self)
   free(self);
 }
 
+// Convert Unix timestamp (seconds since 1970) to 1601-based timestamp
+void
+convert_unix_to_1601(int64_t *timestamp)
```

Comment:
I think it is more clear to return the "new" timestamp instead of passing in a pointer. Then you can use the function inside another function like

```
convert_1601_to_unix(convert_unix_to_1601(timestamp));
```
What do you think?